packages/bull-vault/src/BullStrategy.sol

```diff
@@ -126,7 +126,7 @@ contract BullStrategy is ERC20, LeverageBull, UniBull {
         IERC20(_tokenIn).transfer(_pool, _amountToPay);
     }
 
-    function _getCrabVaultDetails() internal view returns (uint256, uint256) {
+    function getCrabVaultDetails() internal view returns (uint256, uint256) {
```

Comment:
Why removing the `_`, we usually write internal/private functions with `_` in the beginning

Reply:
Was considering making it external func so I can use it in `FlashBull`. Should we just rewrite the function in `FlashBull`?

Reply:
Let's keep `_getCrabVaultDetails()` internal so it can be called from contract that inherit BullStrategy and it will be cheaper, and have a separate `getCrabVaultDetails()` external that can be called from other contract where:
```
getCrabVaultDetails() {
  _getCrabVaultDetails()
}
```